cmd/bootstrap/cmd/finalize.go

```diff
@@ -212,6 +212,12 @@ func finalize(cmd *cobra.Command, args []string) {
 		log.Fatal().Err(err).Msg("the generated root snapshot is invalid")
 	}
 
+	// validate the generated root snapshot QCs
+	err = badger.IsValidRootSnapshotQCs(snapshot)
```

Comment:
could we move `IsValidRootSnapshotQCs` into `IsValidRootSnapshot`?

Reply:
We cannot because `IsValidRootSnapshot` is used in ~130 tests, moving it will break every test or require us to generate a completely valid QC instead of fixtures.